assets/styles/_responsive.scss

```diff
@@ -267,6 +292,12 @@
       margin-top: -97vh;
     }
   }
+
+  .header-carousel .overlay {
```

Comment:
let's refrain from changing heights and widths from now. let's focus on the fonts and typography.